readthedocs/doc_builder/exceptions.py

```diff
@@ -76,3 +76,7 @@ class MkDocsYAMLParseError(BuildEnvironmentError):
         'The "{config}" config from your MkDocs YAML config file has to be a '
         'a list of relative paths.',
     )
+    NOT_FOUND = ugettext_noop(
+        'A configuration file was not found. '
+        'Make sure you have a "mkdocs.yaml" file in your repository.',
```

Comment:
Not sure if it matters, but official MkDocs uses `mkdocs.yml` to refer this file. I suppose it's better to follow their convention here.

Reply:
@humitos Fixed. :)